projects/sandbox/generate_glitches/generate_glitches.py

```diff
@@ -254,12 +254,11 @@ def omicron_main_wrapper(
     omicron_main(omicron_args)
 
 
-@typeo
+@scriptify
 def main(
     snr_thresh: float,
     start: int,
     stop: int,
-    test_stop: int,
```

Comment:
Was there a reason for this? This was included to allow us to run omicron on the testing set.

Reply:
Oh, I see. Yeah, I don't know exactly what I've done, but I've screwed up my version of the code somehow. My version had the `test_stop` argument in the function definitions, but not in the `generate_glitches` section of the `pyproject.toml`, and not anywhere in the rest of the `generate_glitches` script, so I figured it wasn't supposed to be there.

I'll add it back in.

Reply:
Definitely my fault. I likely didn't add it in the `generate_glitches` config